package.json

```diff
@@ -106,5 +104,23 @@
     "sinon": "^1.7.3",
     "snyk": "^1.83.0",
     "watchify": "^3.7.0"
+  },
+  "commitlint": {
```

Comment:
When/how does this run?

Reply:
In the `commitmsg` husky hook : 
https://github.com/segmentio/analytics.js-core/blob/1fb74d18bcf6637f58ba74dcaa7d3f25b75a3812/package.json#L20

Or manually : `yarn commitlint --from HEAD~2`